src/sentry/testutils/factories.py

```diff
@@ -1236,17 +1237,16 @@ def create_doc_integration_features(
         )
 
     @staticmethod
-    @assume_test_silo_mode(SiloMode.REGION)
+    @assume_test_silo_mode(SiloMode.CONTROL)
     def create_doc_integration_avatar(doc_integration=None, **kwargs) -> DocIntegrationAvatar:
         if not doc_integration:
             doc_integration = Factories.create_doc_integration()
-        photo = File.objects.create(name="test.png", type="avatar.file")
+        photo = ControlFile.objects.create(name="test.png", type="avatar.file")
         photo.putfile(io.BytesIO(b"imaginethiswasphotobytes"))
 
-        with assume_test_silo_mode(SiloMode.CONTROL):
-            return DocIntegrationAvatar.objects.create(
-                doc_integration=doc_integration, avatar_type=0, file_id=photo.id
-            )
+        return DocIntegrationAvatar.objects.create(
+            doc_integration=doc_integration, avatar_type=0, control_file_id=photo.id
+        )
```

Comment:
This was wrong before. DocIntegrationAvatar is a control silo model and I missed that this was using file_id before.